src/components/CreateOffer/OfferForm.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { Formik, Form, Field } from "formik";
+import { number, object, string, date } from "yup";
+import { Button, Box, TextField, FormGroup } from "@material-ui/core";
+import DatePicker from "./DatePicker";
+import InputError from "./InputError";
+
+const min = new Date();
+const minDate = `${min.getMonth() + 1}/${min.getDate()}/${min.getFullYear()}`;
+const max = new Date();
+
+const initialValues = {
+  startDate: null,
+  endDate: null,
+  pricePerHour: null,
+  address: null,
+  qualifications: null
+};
+const validationSchema = object({
+  startDate: date()
+    .default(new Date(min))
+    .min(min, `Start date should be equal or later than ${minDate}`)
+    .required("Start date is required"),
+  endDate: date()
+    .default(new Date(max))
+    .min(min, `End date should be equal or later than ${minDate}`)
+    .required("End date is required"),
+  pricePerHour: number()
+    .required("Price per hour is required")
+    .min(1, "You can not choose a price less than 1$ per hour"),
+  address: string()
+    .min(5)
+    .required("Your address is required"),
+  qualifications: string().min(20)
+});
+
+const OfferForm = () => {
+  return (
+    <Formik
+      validationSchema={validationSchema}
+      initialValues={initialValues}
+      onSubmit={async ({ values }) => {
```

Comment:
why don't you make a separate function to handle submission.??

Reply:
This is just a dummy behavior until integrating with the backend